website/docs/core-concepts/connecting-to-data-sources/README.md

```diff
@@ -1,5 +1,41 @@
 # Connecting to Datasource
 
+## **Outline**
+
+**Introduction** 
+
+**Prerequisites​**
```

Comment:
I think this section is unnecessary unless there is some pre-requisite to connecting to a data source.  Creating an account on Appsmith is a known thing and does not need to be mentioned here

Reply:
note.